From b569e3cf2fd855de1eda958e4478cb564a6f2490 Mon Sep 17 00:00:00 2001
From: Ruiqiang Hao <Ruiqiang.Hao@windriver.com>
Date: Thu, 7 Nov 2024 17:23:04 +0800
Subject: [PATCH] Revert "arm: dts: imx6q-sabresd: Enable imx6q pcie"

This reverts commit ffb7412ca5fbc3267803eff102a40fef061ee9f8

There is a separate DTS specifically responsible for enabling PCIe,
PCIe does not need to be enabled in this DTS.

Upstream-Status: Pending

Signed-off-by: Ruiqiang Hao <Ruiqiang.Hao@windriver.com>
---
 arch/arm/boot/dts/nxp/imx/imx6q-sabresd.dts | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/arch/arm/boot/dts/nxp/imx/imx6q-sabresd.dts b/arch/arm/boot/dts/nxp/imx/imx6q-sabresd.dts
index b4463f952831..ca8c7d1a29cb 100644
--- a/arch/arm/boot/dts/nxp/imx/imx6q-sabresd.dts
+++ b/arch/arm/boot/dts/nxp/imx/imx6q-sabresd.dts
@@ -53,7 +53,3 @@ &ipu1_csi1_from_mipi_vc1 {
 	clock-lanes = <0>;
 	data-lanes = <1 2>;
 };
-
-&pcie {
-	status = "okay";
-};
-- 
2.46.2

